### PR TITLE
add links to websites and man pages, change man urls to .org

### PR DESCRIPTION
**Pull request by bobertlo**

### CODE REVIEW

**Comment by Vaelatern** [comment]
I like this idea.

**Comment by the-maldridge** [comment]
Please change the commit message to read "various: add and update links"

**Comment by bobertlo** [comment]
Sure. Done

File: src/config/services/logging.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 The default installation comes with no syslog daemon, there are
 different implementations available.
 
-`socklog` is the implementation from the
+[socklog](http://smarden.org/socklog/) is the implementation from the
 [runit(8)](https://man.voidlinux.eu/runit.8) author and Void Linux
```
Comment (CameronNemo) [inline src/config/services/logging.md]: missed a .eu here

Comment (bobertlo) [reply src/config/services/logging.md]: Oops! Caught a few more. The only ones left are in installation/downloading.md and are fixed upstream.

**Pull request merged**